code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/identity/TokenResourceManager.java

```diff
@@ -41,7 +46,20 @@
     @Inject
     ExternalServiceAuthProvider externalAuthProvider;
 
+    @Inject
+    TokenService tokenService;
+
+    @Inject
+    TokenAuthLookup tokenAuthLookup;
+
+    @Inject
+    AuthDao authDao;
+
+    @Inject
+    AccountDao accountDao;
+
     private List<TokenCreator> tokenCreators;
+    private static final DynamicBooleanProperty LIMIT_CONCURRENT_SESSIONS = ArchaiusUtil.getBoolean("api.auth.limit.concurrent.sessions");
```

Comment:
yeah, limit kind of sounds like you can set the "number" of concurrent sessions allowed to say "2" or "3" etc. Restrict or Disable might be better.